### ISSUE TITLE
Where is it masked? (In the mask_loss branch)

### ISSUE THREAD

**Issue by gesen2egee** [kohya-ss/sd-scripts#1172]
In doc

```
Masked loss
`train_network.py, sdxl_train_network.py and sdxl_train.py now support the masked loss. --masked_loss option is added.
`
NOTE: train_network.py and sdxl_train.py are not tested yet.

ControlNet dataset is used to specify the mask. The mask images should be the RGB images. The pixel value 255 in R channel is treated as the mask (the loss is calculated only for the pixels with the mask), and 0 is treated as the non-mask. See details for the dataset specification in the [LLLite documentation](https://github.com/kohya-ss/sd-scripts/blob/masked-loss/docs/train_lllite_README.md#preparing-the-dataset).
```

But from what I see in the code, it seems that white should be unmasked, and black should be masked. 
Which makes more sense.
```
                    if args.masked_loss:
                        # mask image is -1 to 1. we need to convert it to 0 to 1
                        mask_image = batch["conditioning_images"].to(dtype=weight_dtype)[:, 0].unsqueeze(1)  # use R channel

                        # resize to the same size as the loss
                        mask_image = torch.nn.functional.interpolate(mask_image, size=loss.shape[2:], mode="area")
                        mask_image = mask_image / 2 + 0.5
                        loss = loss * mask_image
```
Is it possible that the description is written incorrectly?
